rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/packages/PackageController.java

```diff
@@ -224,8 +224,6 @@ public ResponseEntity<?> getPackagesForUser(
             @RequestParam(value = "packageManager", required = false) String packageManager,
             @Parameter(description = "Get all details of the package.")
             @RequestParam(value = "allDetails", required = false) boolean allDetails,
-            @Parameter(description = "If true, packages will be fetched by name exactly matching the search input.")
-            @RequestParam(value = "exactMatch", required = false) boolean isExactMatch,
```

Comment:
Any reason to remove this parameter?

Reply:
Because this parameter was never used in this endpoint.